src/SprykerEco/Zed/Unzer/Persistence/UnzerEntityManager.php

```diff
@@ -157,4 +138,77 @@ protected function saveOrDeleteMerchantUnzerParticipantEntity(SpyMerchantUnzerPa
 
         return $merchantUnzerParticipantEntity;
     }
+
+    /**
+     * @param \Generated\Shared\Transfer\UnzerCredentialsTransfer $unzerCredentialsTransfer
+     *
+     * @return \Generated\Shared\Transfer\UnzerCredentialsTransfer
+     */
+    public function createUnzerCredentials(UnzerCredentialsTransfer $unzerCredentialsTransfer): UnzerCredentialsTransfer
```

Comment:
Maybe we should return some response transfer in these CUD methods to identify errors?

Reply:
ResponseTransfer is returned by UnzerCredentialsCreator